server/notification-providers/flashduty.js

```diff
@@ -90,6 +90,7 @@ class FlashDuty extends NotificationProvider {
             options.client_url = baseURL + getMonitorRelativeURL(monitorInfo.id);
         }
 
+        options = this.getAxiosConfigWithProxy(options);
```

Comment:
I am not sure if this works the same way.

Because I cannot test this integration, let's either (preferred) move this to the `axios.post(url, data, config)` API that we are using elsewhere or remove the proxy feature from these notification providers.